src/main/java/slimeknights/tconstruct/library/registration/BlockDeferredRegister.java

```diff
@@ -153,4 +153,25 @@ public FenceBuildingBlockObject registerFenceBuilding(final String name, final B
     }
     return new EnumObject<>(map);
   }
+
+  /**
+   * Registers a block with slab, stairs, and walls
```

Comment:
Wrong javadoc start, this is `registerEnum`. You should also be sure to explain why this is different from the other `registerEnum`

Reply:
True, but this is the same javadoc as you originally had for the other
registerEnum. I should have caught that.

Reply:
Oh is it? Huh, my bad. In any case, both javadocs need an update to say why they are different from each other. One prefixes, one suffixes.